Makefile

```diff
@@ -70,3 +70,59 @@ libscanhttp-restart:
 .PHONY: libvulnhttp-restart
 libvulnhttp-restart:
 	$(docker-compose) up -d --force-recreate libvulnhttp
+
+.PHONY: podman-dev-up
+podman-dev-up:
+	podman pod create\
+		--publish 5434\
+		--publish 8080\
+		--publish 8081\
+		--name claircore-dev
+	podman create\
+		--pod claircore-dev\
+		--name claircore-database\
+		--env POSTGRES_USER=claircore\
+		--env POSTGRES_DB=claircore\
+		--env PGPORT=5434\
+		--expose 5434\
+		--volume $$(git rev-parse --show-toplevel)/internal/scanner/postgres/bootstrap.sql:/docker-entrypoint-initdb.d/libscan-bootstrap.sql:z\
+		--volume $$(git rev-parse --show-toplevel)/internal/vulnstore/postgres/bootstrap.sql:/docker-entrypoint-initdb.d/libvuln-bootstrap.sql:z\
+		--health-cmd "pg_isready -U claircore -d claircore"\
```

Comment:
My hunch is this is not going to block the other containers like 

```
	$(docker) exec -it claircore_claircore-db_1 bash -c 'while ! pg_isready; do echo "waiting for postgres"; sleep 2; done'
```

does - is that okay?

Reply:
You're right, I'll add a thing